src/porepy/fracs/simplex.py

```diff
@@ -634,6 +647,23 @@ def __merge_domain_fracs_2d(dom, frac_p, frac_l):
 
 def __find_intersection_points(lines):
```

Comment:
While we are at it, I think we should change this and the previous function name to single underscore.

Reply:
only this function?